native/src/modules/endpoint/sagas/watchFetchNews.js

```diff
@@ -15,16 +15,17 @@ import loadTunews from './loadTunews'
 import loadTunewsLanguages from './loadTunewsLanguages'
 import loadTunewsElement from './loadTunewsElement'
 import { LOCAL_NEWS_TYPE } from 'api-client/src/routes'
+import loadLanguages from './loadLanguages'
 
 const TUNEWS_FETCH_COUNT_LIMIT = 20
 const FIRST_PAGE_INDEX = 1
 
 export function* fetchNews(dataContainer: DataContainer, action: FetchNewsActionType): Saga<void> {
-  const { city, language, newsId, key, type } = action.params
+  const { city, language, newsId, key, type, criterion } = action.params
   try {
     const isLocalNews = type === LOCAL_NEWS_TYPE
 
-    const languages = yield call(dataContainer.getLanguages, city)
+    const languages = yield call(loadLanguages, city, dataContainer, criterion.forceUpdate)
```

Comment:
Fixes peeking to news. Otherwise this would throw an error as languages have not been loaded for the peeking city.

Reply:
Are you sure forceUpdate is correct here? Not sure what it actually does though.

Reply:
Yes, it is. If `criterion.forceUpdate` is true here, languages are fetched even if there are already languages available offline.